### ISSUE TITLE
Issue detecting commit SHA

### ISSUE THREAD

**Issue by johnwheal** [codecov/codecov-action#190]
Since release v1.1.0 I am getting the following error:

`->  Issue detecting commit SHA. Please run actions/checkout with fetch-depth > 1 or set to 0`

**Comment by ktomk** [comment]
Already triaged in codecov-bash  https://github.com/codecov/codecov-bash/issues/397 , a new action release with this revision would be nice, on Github I see  Bash-20201223-5ad85e3 since days and could not yet confirm good.

**Comment by thomasrockhu** [comment]
This has been released. If you are still experiencing the issue, you will likely need to set `fetch_depth != 1` in your `checkout` stage.

**Comment by ktomk** [comment]
I can now confirm:

old: Bash-20201223-5ad85e3
fix: Bash-20201231-23d4836

Thanks @thomasrockhu

**Comment by silva96** [comment]
if you are using github actions with the codecov bash uploader

```
- uses: actions/checkout@v2
        with:
        fetch-depth: 2
```
